salon_creator/lib/firebase/database.dart

```diff
@@ -1,4 +1,5 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
 import 'package:salon_creator/models/salon.dart';
 
 final salonsRef = FirebaseFirestore.instance.collection('salons').withConverter(
```

Comment:
It is a good idea to keep the **salonsRef** parameter as a property in the class and initialize it in the constructor. By doing this, **salonsRef** will be created only when an instance of the **DbHandler** is created. Currently, this will statically be created all the time and can be optimized!

```
class DbHandler {
  var salonsRef;
  DbHandler() {
    salonsRef = ...
  }
}
```